std/utf.d

```diff
@@ -1164,8 +1164,10 @@ package template codeUnitLimit(S)
  * Returns:
  *      decoded character
  */
-private dchar decodeImpl(bool canIndex, UseReplacementDchar useReplacementDchar = UseReplacementDchar.no, S)(auto ref S str, ref size_t index)
-    if (is(S : const char[]) || (isInputRange!S && is(Unqual!(ElementEncodingType!S) == char)))
+private RV decodeImpl(bool canIndex, UseReplacementDchar useReplacementDchar, RV = dchar, S)
```

Comment:
I not so sure about the name RV. I guess it is supposed to mean ReturnValue, but isn't CharType clearer?